Server/htdocs/AppController/commands_RSM/api/v2/items/getTypes.php

```diff
@@ -0,0 +1,68 @@
+<?php
+//***************************************************************************************
+// Description:
+//    Get one, multiple or all item types and its associated propertyIDS + NAME
+// REQUEST BODY (JSON OBJECT):
+//  EXAMPLE 1:
+//    -Use the endpoint without any param or body to obtain all of them
+//
+//  EXAMPLE 2
+//    - Add a query param with the specific itemtypeIDs
+//            ?ID=6,7,8
+//***************************************************************************************
+require_once "../../../utilities/RStools.php";
+require_once "../../../utilities/RSMverifyBody.php";
+setAuthorizationTokenOnGlobals();
+checkCorrectRequestMethod('GET');
+
+require_once "../../../utilities/RSdatabase.php";
+require_once "../../../utilities/RSMitemsManagement.php";
+require_once "../../api_headers.php";
+
+// Definitions
+$clientID = getClientID();
+$RStoken = getRStoken();
+$RSuserID = getRSuserID();
+
+// Check if 'ID' is not set or is empty in the $_GET parameters
+
+if (!isset($_GET['ID']) && empty($_GET['ID'])) {
```

Comment:
Esta validación no tiene sentido, si no está definido, por que además validamos que esté vacío? puede que sea un "or" en vez de un "and"?